### ISSUE TITLE
Interactive maps server

### ISSUE THREAD

**Issue by kymni** [akvo/akvo-provisioning#213]
php 5+ server to host interactive maps

**Comment by iperdomo** [comment]
@kymni are those _all_ the requirements for the server? Is there a database involved? how about DNS configuration? Do we need a special subdomain setup?

**Comment by kymni** [comment]
Only one app (though not active) requires a local database server so a MySQL server can be included for it. All the rest of the apps are integrated with cartodb which takes care of all data management. 
For subdomain, maybe "interactive-maps". The individual apps will then follow a directory structure

**Comment by iperdomo** [comment]
@kymni is MySQL really a requirement? Can we use PostgreSQL instead?

**Comment by kymni** [comment]
@iperdomo yes we can use PostgreSQL instead. Regarding the tech stack, am work on a detailed list containing the current setup, dependencies, and configurations which I'll then post to this thread. I'll however most likely post it on Monday after I get back to Nairobi

**Comment by mato74** [comment]
@kymni where does this stand? Any open issues we need to look at?

**Comment by kymni** [comment]
@mato74 this one's on me, refocusing on it to get it out the way soonest possible

**Comment by mato74** [comment]
@kymni please hold on spending time on this issue as we investigate further what the best next steps are. Especially as there is a large (100%?) overlap between the interactive maps and Akvo Lumen development. I will plan a meet with you, Ivan, Lars and Mark.

**Comment by mtwestra** [comment]
In short, the strategy has been to develop Lumen to the point where we can reimplement the interactive maps there. Let's discuss this afternoon if this is still the best path forward.
